charts/aks-operator/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+aksOperator:
+  image:
+    repository: rancher/aks-operator
+    tag: v1.0.1-rc1
```

Comment:
I think it's better to set this to a nonsense value to help indicate that it is overwritten. Something like `999` or `v0.0.0`.

Reply:
done